openedx/features/partners/views.py

```diff
@@ -38,3 +38,22 @@ def register_user(request, slug):
         return views.Give2AsiaRegistrationView.as_view()(request, partner=partner)
     except ImportError:
         raise Http404('Your partner is not properly registered')
+
+
+@require_http_methods(["POST"])
+@sensitive_post_parameters('password')
+def login_user(request, slug):
+    """
+    This is general login view, for users of all partners
+    :param request.
```

Comment:
Lets make param doc consistent, we can update it to `:param request: The Django request.`

Reply:
Sure